__extensions_index.ts

```diff
@@ -5,7 +5,16 @@ import { CustomFieldsModule } from "./custom-fields";
 import { CommandsExtenderModule } from "./commands-extender";
 import { ToolbarExtenderModule } from "./toolbar-extender";
 
+/**
+ * The entry point of the extensions. Each extension needs to have exactly one export of the Extensions interface.
+ * and it needs to be placed in the __extensions_index file.
+ * Here all of the angular modules are returned and are loaded into the main module.
```

Comment:
*Here all of the angular modules are...*

> maybe it is unclear what is extension module and angular module, see if this looks ok:

*Here all of **the extensions as** angular modules are...*